crates/solidity/outputs/cargo/crate/src/backend/ir/ast/ast_nodes.generated.rs

```diff
@@ -0,0 +1,6549 @@
+// This file is generated automatically by infrastructure scripts. Please don't edit by hand.
+
+//
+// Sequences:
+//
+
+pub type SourceUnit = Rc<SourceUnitStruct>;
```

Comment:
In a line similar to Omar's comment above, do we need to wrap these types in an `Rc`? functions like `members` below could directly return the struct.

Reply:
Yeah, as I replied above, when you need to cross to Wasm you'll need an `Rc` anyway, but for the Rust API it may make sense. I'll explore the alternative.